lib/combine_pdf/page_methods.rb

```diff
@@ -63,15 +63,16 @@ def inject_page(obj, top = true)
       # injecting each of the values in the injected Page
       res = resources
       obj.resources.each do |key, new_val|
```

Comment:
This part of the code, which I wrote, is slower then writing `obj.resources.keys.each` and it provides less flexibility about the handling of the data... Maybe the whole thing should be revised.